### PR TITLE
Add post on Html.Lazy

### PR DESCRIPTION
**Pull request by Arkham**
👯

### CODE REVIEW

File: content/blog/performant-elm-html-lazy/index.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+read about [how to tell what your browser is really
+doing](../performant-elm) and [how Html.Keyed can help you help Elm to be
+faster](../performant-elm-html-keyed).
+
+Today we are talk about being lazy. As an Italian, I know a thing or two
+about being lazy. I'm not talking "Oh I spent Sunday binge-watching Derry
+Girls" lazy, but more something like ["I got caught with 400kg of undelivered
+mail in my flat"](https://www.theguardian.com/world/2018/apr/06/former-postman-found-with-400kg-of-undelivered-mail-in-italy) sort of lazy.
+
+So [the Elm Guide](https://guide.elm-lang.org/optimization/lazy.html) does a fantastic job at explaining what `Html.Lazy` does. I really recomment going there and reading about it. Anyway, the main idea of it is this function:
```
Comment (quelledanielle) [inline content/blog/performant-elm-html-lazy/index.md]:
```suggestion
So [the Elm Guide](https://guide.elm-lang.org/optimization/lazy.html) does a fantastic job at explaining what `Html.Lazy` does. I really recommend going there and reading about it. Anyway, the main idea of it is this function:
```

**Comment by Arkham** [comment]
Great point, I will add a section about that :)

**Comment by Arkham** [comment]
@quelledanielle what do you think about this now? :)

**Comment by quelledanielle** [comment]
@Arkham Looks great! 🎉

**Pull request merged**